services/post_service.py

```
@@ -0,0 +1,100 @@
+from models.post import Post, PostInternal, PostQuery
+from fastapi import HTTPException
+from repositories.posts_repo import get_post, save_post
+from datetime import datetime
+
+def write_posts(data: Post, current_user: dict):
+    posts = get_post()
+
+    if any(u["title"] == data.title for u in posts):
+        raise HTTPException(400, "이미 존재하는 게시물 타이틀")
+
+    post_created_at = datetime.now()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)